bin/python/ripple/util/test_Sign.py

```diff
@@ -59,13 +52,17 @@ def test_make_manifest(self):
             '\xef*\x97\x16n<\xa6\xf2\xe4\xfb\xfc\xcd\x80P[\xf1s\x06verify')
 
     def test_sign_manifest(self):
+        ssk, spk = Sign.make_ecdsa_keypair(self.urandom)
```

Comment:
There's a README.md in the python directory. The incantation is `python -m unittest discover`. Point taken though about making that more obvious.